tests/camera/test_camera.py

```diff
@@ -19,23 +20,25 @@
 @pytest.mark.datafiles(
     join(CAM_DIR, "raw.jpg"),
 )
-def test_normal_use(datafiles):
-    sett.set(
-        ["mrbeam", "mock", "img_static"],
-        str(datafiles / "raw.jpg"),
-        force=True,
-    )
-    worker = MrbPicWorker()
-    with MrbCamera(worker, shutter_speed=0.5) as cam:
-        cam.capture()
-        cam.async_capture()
-        cam.wait()
-        assert worker.count == 2
-        assert cam.lastPic() is not None
+def test_normal_use(datafiles, mrbeam_plugin):
+    def mock_get(*args, **kwargs):
+        if args[0] == ["mrbeam", "mock", "img_static"]:
+            return str(datafiles / "raw.jpg")
+
+    with patch(
```

Comment:
Maybe we could create a fixture with the MrbCamera (similar to what we do with mrbeam_plugin)?

Reply:
I was trying this but as I need to get the datafiles in there and these are parameterized, it was not possible